packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

```diff
@@ -25,47 +25,42 @@ Okta provides a very rich [Authentication API](/docs/reference/api/authn/) to va
 - [Retrieving a session cookie by visiting a session redirect link](/docs/guides/session-cookie/#retrieving-a-session-cookie-by-visiting-a-session-redirect-link)
 - [Retrieving a session cookie by visiting an application embed link](/docs/guides/session-cookie/#retrieving-a-session-cookie-by-visiting-an-application-embed-link)
 
-> **Session Tokens** are secrets and should be protected at rest as well as during transit. A session token for a user is equivalent to having the user's actual credentials.
+> **Session tokens** are secrets and should be protected at rest and during transit. A session token for a user is equivalent to having the user's actual credentials.
 
-## Getting Started
+## Get started
 
 Explore the Sessions API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9f729bbb060e6bac09fa)
 
-## Session Operations
-
-### Create Session with Session Token
+## Session operations
 
+### Create session with a session token
 
 <ApiOperation method="post" url="/api/v1/sessions" />
 
-Creates a new session for a user with a valid session token. Use this API if, for example, you want to set the session cookie yourself instead of allowing Okta to set it, or want to hold the session ID in order to delete a session via the API instead of visiting the logout URL.
+Creates a new session for a user with a valid session token. Use this API if, for example, you want to set the session cookie yourself instead of allowing Okta to set it, or you want to hold the session ID to delete a session through the API instead of visiting the logout URL.
```

Comment:
do a find and replace for session in this doc - anything that is referring to the Session object should be Session, not session. For example, for this sentence: 

Creates a new Session for a user with a valid session token. Use this API if, for example, you want to set the Session cookie yourself instead of allowing Okta to set it, or you want to hold the session ID to delete a Session through the API instead of visiting the logout URL.